src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackValidations.java

```diff
@@ -57,6 +57,15 @@ private CallbackValidations() {
             .orElse(invalid("Missing caseType"));
     }
 
+    @Nonnull
+    public static Validation<String, String> hasFormType(CaseDetails theCase) {
```

Comment:
It would be best to unit-test this method.

Reply:
I added a unit test in `CallbackValidationsTest`

Reply:
Sorry, I'm blind